sample/package.json

```diff
@@ -1,25 +1,26 @@
 {
-  "name": "sample",
-  "description": "sample package for testing hardhat-multibaas-plugin",
+  "name": "hardhat-multibaas-plugin-sample-v3",
+  "private": true,
   "version": "1.0.0",
-  "author": "Curvegrid",
-  "license": "MIT",
+  "description": "Sample Hardhat v3 project for hardhat-multibaas-plugin",
+  "type": "module",
   "engines": {
     "node": ">=24.0.0"
   },
   "scripts": {
-    "test": "npx hardhat test",
-    "deploy:metacoin": "HARDHAT_NETWORK=development npx hardhat run scripts/deploy-metacoin.ts",
-    "deploy:greeter": "HARDHAT_NETWORK=development npx hardhat run scripts/deploy-greeter.ts",
-    "deploy:link:greeter": "HARDHAT_NETWORK=development npx hardhat run scripts/deploy-then-link-greeter.ts",
-    "deploy:proxy:greeter": "HARDHAT_NETWORK=development npx hardhat run scripts/deploy-proxied-greeter.ts",
+    "deploy:metacoin": "HARDHAT_NETWORK=development npx hardhat ignition deploy ignition/modules/MetaCoinModule.ts",
+    "deploy:greeter": "HARDHAT_NETWORK=development npx hardhat ignition deploy ignition/modules/GreeterModule.ts",
+    "deploy:link:greeter": "HARDHAT_NETWORK=development npx hardhat ignition deploy ignition/modules/LinkedGreeterModule.ts",
+    "deploy:proxy:greeter": "HARDHAT_NETWORK=development npx hardhat ignition deploy ignition/modules/ProxiedGreeterModule.ts",
```

Comment:
Given this is a development network, perhaps add `HARDHAT_IGNITION_CONFIRM_DEPLOYMENT=false` to these lines to skip the prompt?